themes/default/layouts/migrate/terraform.html

```diff
@@ -130,7 +130,7 @@ <h5>Adopt existing resource state</h5>
                             <h5>Coexist with Terraform state</h5>
                             <div class="flex flex-col lg:flex-row justify-center items-center mt-4">
                                 <div class="my-2 mx-2 w-full lg:w-auto">
-                                    <a href="{{ relref . "/blog/using-terraform-remote-state-with-pulumi" }}" class="w-full block md:w-auto md:inline-block btn-secondary"
+                                    <a href="{{ relref . "/blog/converting-full-terraform-programs-to-pulumi" }}" class="w-full block md:w-auto md:inline-block btn-secondary"
```

Comment:
@gthuang, I think line 101 above should be updated to link to the new blog post and this one should stay the same.